### ISSUE TITLE
discv5 issue in dockerized environment

### ISSUE THREAD

**Issue by twoeths** [ChainSafe/lodestar#2286]
**Describe the bug**

We have 0 peers in dockerized environment, note that it's not an issue if we run the app directly from node `node lib/index.js beacon`. From the log:
```
2021-03-29T08:00:00.483Z discv5:service Sending lookup. Id: 23, Iteration: 1, Node: 73b3df542a85283fb4633bc1239077ef31326a528d9be476b961bc9dc84ba90f
2021-03-29T08:00:00.483Z discv5:sessionService No session established, sending a random packet to: 73b3df542a85283fb4633bc1239077ef31326a528d9be476b961bc9dc84ba90f on /ip4/54.65.172.253/udp/9000
2021-03-29T08:00:00.483Z discv5:service Sending lookup. Id: 23, Iteration: 1, Node: 6dc22b7deb7ca7f937f3e239629c9666cf866971df45faea4335b596e535cfbc
2021-03-29T08:00:00.483Z discv5:service Sending request to node: 6dc22b7deb7ca7f937f3e239629c9666cf866971df45faea4335b596e535cfbc failed: error: Session is being established, request failed
2021-03-29T08:00:01.479Z discv5:sessionService Session couldn't be established with node: f7fa00ba76b8e33caae49ba504b81a2389a963a7c990ec722c085ec663ac2492 at /ip4/54.178.44.198/udp/9000
2021-03-29T08:00:01.484Z discv5:sessionService Session couldn't be established with node: 73b3df542a85283fb4633bc1239077ef31326a528d9be476b961bc9dc84ba90f at /ip4/54.65.172.253/udp/9000
2021-03-29T08:00:01.484Z discv5:service Lookup Id: 23 finished, 0 total found
2021-03-29T08:00:01.484Z discv5:service Starting a new lookup. Id: 24
2021-03-29T08:00:01.486Z discv5:service Sending lookup. Id: 24, Iteration
```

**Expected behavior**

We should be able to have peers in dockerized environment

**Comment by dapplion** [comment]
I've changed the node version on the docker image and works fine now

- `node:14-alpine` -> `v14.16.0`: No peers 
- `node:12-alpine` -> `v12.21.0`: peers as normal

**Comment by wemeetagain** [comment]
Should be fixed with #2290, will keep the issue open for confirmation

**Comment by twoeths** [comment]
Confirm that it's working fine now.
